lib/xcpretty/formatters/simple.rb

```diff
@@ -8,11 +8,11 @@ class Simple < Formatter
     PASS = "✓"
     FAIL = "✗"
     PENDING = "⧖"
+    COMPLETION = "▸"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.